charts/incubator/linkwarden/values.yaml

```diff
@@ -0,0 +1,71 @@
+image:
+  repository: ghcr.io/linkwarden/linkwarden
+  pullPolicy: IfNotPresent
+  tag: v2.2.1@sha256:c948cdcf683ec7b6c634eb2a0e184e270e167401d41ef8e82f7ff64c61f39f9c
+
+config:
+  pagination_take_count: 20
+  storage_folder: "/data"
+  autoscroll_timeout: 30
+  disable_registration: false
+  archive_limit: 5
+  smtp:
+    enabled: false
+    from: ""
+    server: ""
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+    runAsUser: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 2985
+        targetPort: 3000
+
+workload:
+  main:
+    podSpec:
+      containers:
+        main:
+          env:
+            DATABASE_URL:
+              secretKeyRef:
+                name: cnpg-main-urls
+                key: std
+            POSTGRES_PASSWORD:
+              secretKeyRef:
+                name: cnpg-main-user
+                key: password
+            NEXTAUTH_SECRET:
+              secretKeyRef:
+                name: linkwarden-secrets
+                key: NEXTAUTH_SECRET
+            NEXTAUTH_URL: "http://localhost:3000/api/v1/auth"
+            PAGINATION_TAKE_COUNT: "{{ .Values.config.pagination_take_count }}"
+            STORAGE_FOLDER: "{{ .Values.config.storage_folder }}"
```

Comment:
Is there a reason to allow user to change this? If not let’s hardcode it here.

Reply:
storage folder? Probably not. I'll change it